apps/web/src/pages/EventPage/EventPage.tsx

```diff
@@ -107,98 +107,172 @@ export default function EventPage({
+                      >
+                        Decline
+                      </button>
+                      <button
+                        type="button"
+                        className="ml-3 inline-flex justify-center py-2 px-4 border border-transparent rounded-md shadow-sm text-sm font-medium text-white bg-slate-500 hover:bg-slate-600 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-slate-500"
+                        onClick={handleRemove}
+                      >
+                        Leave
+                      </button>
+                    </div>
+                  </dd>
+                </div>
+              )}
+              {/* render for the host only */}
+              {eventData.hostId === userRecord.uid && (
+                <div className="sm:col-span-2">
+                  <dd className="mt-1 text-sm text-gray-900">
+                    <div id="eventFinalizedButtons">
+                      <button
+                        type="button"
+                        className="ml-3 inline-flex justify-center py-2 px-4 border border-transparent rounded-md shadow-sm text-sm font-medium text-white bg-blue-500 hover:bg-blue-600 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-blue-500"
+                        onClick={handleUpdateGuests}
+                      >
+                        Update Guest List
+                      </button>
+                      <button
+                        type="button"
+                        className="ml-3 inline-flex justify-center py-2 px-4 border border-transparent rounded-md shadow-sm text-sm font-medium text-white bg-red-500 hover:bg-red-600 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-red-500"
+                        onClick={handleDelete}
+                      >
+                        Delete Event
+                      </button>
+                    </div>
+                  </dd>
+                </div>
+              )}
+              <div className="sm:col-span-2">
+                <dt className="text-sm font-medium text-gray-500">Attendees</dt>
+                <dd className="mt-1 text-sm text-gray-900">
+                  <ul
+                    role="list"
+                    className="max-w-sm border border-gray-200 rounded-md divide-y divide-gray-200"
+                  >
+                    {eventGuests.map((guestName) => {
+                      return guestName.status === 'ACCEPTED' ? (
+                        <li className="pr-4 py-3 flex items-center justify-between text-sm">
+                          <div className="w-0 flex-1 flex items-center">
+                            <span className="ml-2 flex-1 w-0 truncate">
+                              {guestName.firstName} {guestName.lastName}
+                            </span>
+                          </div>
+                          {/* render for the host only */}
+                          {/* TODO: allow host to remove guests */}
```

Comment:
this code will be removed if we don't have the functionality for hosts to remove guests from events after they have been finalized

Reply:
leaving this for now